src/peft/tuners/lora/aqlm.py

```diff
@@ -35,13 +35,27 @@ def __init__(
         lora_dropout: float = 0.0,
         init_lora_weights: bool = True,
         use_rslora: bool = False,
+        use_dora: bool = False,
+        lora_bias: bool = False,
```

Comment:
Indeed I considered this. My main reasoning for going with the more generic `lora_bias` was that it leaves the door open for extending this argument in the future. Say, someone finds that LoRA works much better when also adding a bias to LoRA A, then we can adopt this argument to allow this too. Otherwise, we'd have to add a new argument (and we don't want to rename arguments for obvious reasons). LMK what you think of that reasoning.

Reply:
> Otherwise, we'd have to add a new argument (and we don't want to rename arguments for obvious reasons).

I think that would still be preferrable over having a single argument for controlling the bias setup for LoRAs as I think it's still in its infancy. 

Later it if it becomes a common standard to add biases for both LoRA matrices we can deprecate `lora_B_bias` and `lora_A_bias` (if we introduce such an argument) to have a single argument called `lora_bias`. 

This is where I stand but I am not oo opinonated about it.

Reply:
Do we care about reproducibility after upgrading PEFT? Then it seems detrimental to possibly merge control of A and B biases into one flag in the future and they should be separated into two flags from the start.

Otherwise, I think in terms of opportunity cost for experimentation on the user's side having two separate parameters (`lora_bias_A`, `lora_bias_B`) is better. That said, having only one parameter appears to be simpler: let the implementation decide what the current best thing is for adding biases. So if you are just someone who wants to do LoRA best-current-practice it would be helpful to only have one flag. This becomes harder with two flags since there is no obvious 'no bias at all' vs. 'best-practice' setting. If we have simplicity first (and don't care about reproducibility after upgrading) then one parameter is the way to go, I think. What's the stance here?

Ideally there would be another layer of abstraction, a more low-level abstraction, that has two bias parameters and one above that which decides what the best choice is at the moment. I.e. `BaseLoRA(..., lora_bias_A, lora_bias_B) -> LoRA(..., lora_bias)` .

Reply:
To clarify, my idea is that if we want to later add the possibility for a bias for LoRA A, the option would be something like `lora_bias="a"`, or for both, `lora_bias="both"`. We should not change the meaning of `lora_bias=True`, in order to ensure reproducibility, as you mentioned.

If we find that the parameter gets overloaded, we can add the option for a sub-config, so `LoraConfig(..., lora_bias=LoraBiasConfig(bias_a=True, bias_b=True, ...))`.

Reply:
Seems like `lora_bias` should be fine for now.